client/js/components/procedure/admin/InstitutionTagManagement/InstitutionList.vue

```diff
@@ -726,6 +743,55 @@ export default {
       this.currentlySelectedFilterCategories = selectedCategories
     },
 
+    setFilterOptionsFromFilterQuery () {
+      const filterQueryFromStorage = this.getFilterQueryFromLocalStorage()
+      const categoryIdsWithSelectedFilterOptions = Object.keys(filterQueryFromStorage)
+        .filter(id => id.includes('_group'))
+        .map(id => id.replace('_group', ''))
+
+      categoryIdsWithSelectedFilterOptions.forEach(id => {
+        const selectedFilterOptionIds = Object.values(filterQueryFromStorage)
```

Comment:
maybe performance wise, we can move the declaration out of the loop, the `Object.values(filterQueryFromStorage)` would be performed only once.